src/sync/utils.rs

```diff
@@ -72,6 +72,8 @@ macro_rules! request_handler {
 #[macro_export]
 macro_rules! client_request {
     ($self: ident, $ctx: ident, $req: ident, $server: expr, $method: expr, $cres: ident) => {
+        println!("writing message");
```

Comment:
Is this for debugging?

Reply:
Yes, was tracking down an issue, will remove this